src/auth/src/credentials/util/jws.rs

```diff
@@ -14,7 +14,6 @@
 
 use crate::credentials::CredentialError;
 use crate::credentials::Result;
-use derive_builder::Builder;
```

Comment:
Thanks. I think there is an unused dependency in `src/auth/Cargo.toml` now.

Reply:
Still used elsewhere:

https://github.com/googleapis/google-cloud-rust/blob/31c8a5eb590cf8908453ddaa14fc35cfb32220fb/src/auth/src/credentials/service_account_credential.rs#L40